doc/design/oss_backup.md

```diff
@@ -0,0 +1,42 @@
+# Backups using Alibaba Cloud Object Storage Service (OSS)
+
+## Cluster configured with OSS backup
+
+To create a backup in OSS, set `backup.storageType` to `"OSS"`, supply the path (in the format `<oss-bucket-name>/<path-to-backup-object>`) in `oss.path` and provide the Kubernetes secret storing the Alibaba Cloud account credentials to `oss.ossSecret`.  The secret must exist prior to backup creation. Etcd backup operator will create the bucket and object if not found. The field `oss.endpoint` is the target OSS service endpoint where the data is backed up. By default, `http://oss-cn-hangzhou.aliyuncs.com` will be used. If you want to back up the data to other regions, please specify another endpoint from [the list of region endpoints](https://www.alibabacloud.com/help/doc-detail/31837.htm).
+
+
+An example backup manifest would look like:
+
+```yaml
+apiVersion: etcd.database.coreos.com/v1beta2
+kind: EtcdBackup
+metadata:
+  name: etcd-cluster-with-oss-backup
+  namespace: my-namespace
+spec:
+  backupPolicy:
+    ...
+  etcdEndpoints:
+    ...
```

Comment:
Let's keep this example consistent with the examples for other backup object types so it works as is:
```suggestion
    - "http://example-etcd-cluster-client:2379"
```
Also remove lines 17-18 since we don't need periodic backup for this example.